docs/guides/create-app/Reducers.md

```diff
@@ -81,10 +81,12 @@ export default createApp({
 
   component: Root,
 
+  // defining how the app should looks like
```

Comment:
`initialState` is about the initial object that will be passed to reducers, as defined here: https://travix-international.github.io/frint/docs/api/createApp.html#initialstate